### PR TITLE
Adds CameraServer and OpenCV to the plugins

### PR DESCRIPTION
**Pull request by ThadHouse**
Note still needs to be tested, and will need to be updated when we rename cameraserver to cscore

### CODE REVIEW

File: edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml [hunk 1]
```diff
@@ -62,6 +62,8 @@
 		<fileset dir="${userLibs.dir}" includes="*.jar"/>
 		<fileset file="${wpilib.jar}"/>
 		<fileset file="${networktables.jar}"/>
+    <fileset file="${opencv.jar}"/>
+    <fileset file="${cscore.jar}"/>
```
Comment (PeterJohnson) [inline edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml]: Whitespace is different?

File: edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml [hunk 2]
```diff
@@ -112,7 +114,13 @@
...
       <libs.local>
         <fileset id="wpiNativeLibs.local" dir="${wpilib.native.lib}">
-          <include name="**/*.so"/>
+          <include name="libHALAthena.so"/>
+          <include name="libntcore.so"/>
+          <include name="libwpilibJavaJNI.so"/>
+          <include name="libwpiutil.so"/>
+          <include name="libopencv*.so.3.1"/>
+          <include name="libcscore.so"/>
+          <include name="libopencv_java310.so"/>
```
Comment (PeterJohnson) [inline edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml:122]: Should we match the order here to the C++ one, and just add the java-specific JNI ones at the end?

File: pom.xml [hunk 3]
```diff
@@ -48,6 +48,8 @@
                     <allowSnapshots>true</allowSnapshots>
                     <includes>
                         <include>edu.wpi.first.*:*</include>
+                        <include>edu.wpi.*:*</include>
```
Comment (PeterJohnson) [inline pom.xml:50]: Does this also match `edu.wpi.first.*`?  Maybe we should be specific and call out cscore here?

Comment (ThadHouse) [reply pom.xml:50]: We could also just remove the first specific one. I think that should be fine.

File: edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml (hunk 1 above)
Comment (ThadHouse) [reply edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml]: Fixed. That file is a whole mix of spaces and tabs, which is fun.

File: edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml (hunk 2 above)
Comment (ThadHouse) [reply edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/build.xml:122]: Done

**Pull request merged**